src/main/java/io/ebean/enhance/Transformer.java

```diff
@@ -37,6 +40,25 @@
  */
 public class Transformer implements ClassFileTransformer {
 
+  private static String version = "unknown";
+  static {
```

Comment:
I don't think we should do this on cinit (all the time) but instead have it as a method that when invoked reads the pom.properties file.

Reply:
I agree